source/kubernetes/kubectl-plugin-reference.rst

```diff
@@ -0,0 +1,650 @@
+=================================
+MinIO Kubernetes Plugin Reference
+=================================
+
+.. default-domain:: minio
+
+.. contents:: On This Page
+   :local:
+   :depth: 2
+
+The ``kubectl-minio`` plugin wraps the MinIO Kubernetes Operator (``operator``)
+and provides a simple interface through ``kubectl`` for deploying and managing
+MinIO resources. The ``kubectl-minio`` interface does not require familiarity
+with the Kubernetes API, the MinIO ``operator`` API extension, or the creation
+of ``YAML`` specifications. 
+
+The MinIO ``kubectl-minio`` plugin is a part of the 
+:minio-git:`MinIO Kubernetes Operator <operator/tree/master/kubectl-minio>` 
+Github repository.
+
+Installation
+------------
+
+Prerequisites
+~~~~~~~~~~~~~
+
+.. list-table::
+   :width: 100%
+   :widths: 10 90
+
+   * - .. raw:: html
+
+          <input type="checkbox">
+
+     - Kubernetes v1.17.0 or later.
+
+   * - .. raw:: html
+
+          <input type="checkbox">
+   
+     - Persistant Volumes (``PV``) created using the 
+       :doc:`MinIO Direct CSI </kubernetes/direct-csi>` plugin. 
+       
+       The cluster should have one Direct CSI ``PV`` for each planned 
+       :mc-cmd:`volume <kubectl minio tenant create volumes>` in the MinIO
+       tenant. For example, if the MinIO deployment will have 4 servers with 4
+       volumes each, the Kubernetes cluster should have 16 total ``PV`` created
+       using the MinIO Direct CSI plugin.
+
+   * - .. raw:: html
+
+          <input type="checkbox">
+   
+     - A machine with ``kubectl`` installed and configured with access to the
+       target Kubernetes cluster. 
+
+This procedure assumes the default ``kubectl`` context points to the
+Kubernetes cluster in which you want to deploy the MinIO tenant. To
+specify a different context, include the ``--context`` argument to
+``kubectl``. For more information on Kubernetes contexts, see
+:kube-docs:`Organizing Cluster Access using kubeconfig Files
+<concepts/configuration/organize-cluster-access-kubeconfig/>`
+
+1) Download the ``kubectl-minio`` Plugin
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Download the latest ``kubectl-minio`` binary from the MinIO ``operator``
+:minio-git:`github repository <opreator/releases>` and place it into your system
+``PATH``. You may need to use ``chmod`` or an equivalent utility to set the
+binary as executable.
+
+The following command:
+
+- Uses ``wget`` to download the latest stable version of the ``kubectl-minio``
```

Comment:
I guess we do not yet publish the plugin but there is a way to get a URL to the latest always. https://docs.github.com/en/enterprise/2.15/user/articles/linking-to-releases#linking-to-the-latest-release

Reply:
Ah, I'll see how that looks once we publish the plugin, since we'll want to download only the binary (and not a tar.gz of the entire thing...or maybe that's how it will look? Will see once we get there)